examples/jlang-parser/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "jlang-parser"
+version = "0.1.0"
+authors = ["Matt Quinn <matt@mattjquinn.com>"]
+edition = "2018"
+
+[dependencies]
+pest = "2.1.0"
+pest_derive = "2.1.0"
```

Comment:
POSIX defines a line *with* a newline character at the end, so it would be nice to try and keep all new files added to the project like that. [Link to discussion.](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)